packages/auto-drive/package.json

```diff
@@ -34,6 +34,9 @@
   "dependencies": {
     "@ipld/dag-pb": "^4.1.2",
     "blake3": "1.1.0",
+    "blockstore-core": "^5.0.2",
+    "interface-blockstore": "^5.3.1",
+    "interface-store": "^6.0.2",
```

Comment:
`interface-store` seem to only be use for types, if so, it could be moved to the devDependencies